src/include/common/type_utils.h

```diff
@@ -3,16 +3,47 @@
 #include <sstream>
 
 #include "common/exception.h"
+#include "common/string_utils.h"
 #include "common/types/types_include.h"
 
 namespace kuzu {
 namespace common {
 
+// This class is used to cast c++ const char* to c++ primitive types. Should be moved to parser
+// once the csv parser is implemented.
+class StringCastOperation {
```

Comment:
@ray6080 should comment on this as well. I think operation should be reserved for function only. Here you probably wanna move to StringUtils. And we should unify the naming of convert and case, I see different usage   here and for date & timestamp

Reply:
I agree. You can either move them to StringUtils, or rename this to StringCastUtils?

Reply:
renamed to stringCastUtils